src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/configuration/KafkaConfiguration.java

```diff
@@ -16,14 +16,20 @@
 @Configuration
 class KafkaConfiguration {
 
-    @Value("${kafka.group.name}")
-    private String groupName;
+    @Value("${kafka.incoming.consumer.group.name}")
+    private String incomingConsumerGroupName;
+
+    @Value("${kafka.retry.consumer.group.name}")
+    private String retryConsumerGroupName;
 
     @Value("${kafka.broker.address}")
     private String brokerAddress;
 
     @Value("${kafka.consumer.topic}")
-    private String topicName;
+    private String incomingTopicName;
+
+    @Value("${kafka.retry.topic}")
```

Comment:
we could probably do away with a config value here if we just do incomingTopicName + "-retry"